DESCRIPTION

```diff
@@ -50,6 +50,7 @@ VignetteBuilder:
 Config/Needs/check: mrc-ide/epireview
 Config/Needs/website: epiverse-trace/epiversetheme, mrc-ide/epireview
 Config/testthat/edition: 3
+Config/potools/style: explicit
```

Comment:
It seems from {potools} documentation that `explicit` is the preferred style. Could you clarify why you chose this over `base`?

Reply:
I wanted fine control over which strings are translatable at this stage. As you can see, I have targeted only the strings for the print method, but not the messages. `explicit` seems like the logical solution to have this level of control.

Reply:
Thanks